patch/0042-armhf-additional-configs.patch

```diff
@@ -0,0 +1,89 @@
+From a961464da6c88aee4d94af4038c748a1e8a1f412 Mon Sep 17 00:00:00 2001
+From: Antony Rheneus <arheneus@marvell.com>
+Date: Thu, 17 Oct 2019 18:57:48 +0530
+Subject: [PATCH] armhf mandatory configs for boot
+
+---
+ debian/build/build_armhf_none_armmp/.config | 53 ++++++++++++++++-------------
+ 1 file changed, 30 insertions(+), 23 deletions(-)
+
+diff --git a/debian/build/build_armhf_none_armmp/.config b/debian/build/build_armhf_none_armmp/.config
+index e997934..69288eb 100644
+--- a/debian/build/build_armhf_none_armmp/.config
++++ b/debian/build/build_armhf_none_armmp/.config
+@@ -1793,16 +1793,16 @@ CONFIG_PROC_EVENTS=y
+ CONFIG_MTD=y
+ # CONFIG_MTD_TESTS is not set
+ # CONFIG_MTD_REDBOOT_PARTS is not set
+-# CONFIG_MTD_CMDLINE_PARTS is not set
++CONFIG_MTD_CMDLINE_PARTS=y
```

Comment:
Please add to the commit message of *this* patch `patch/0042-armhf-additional-configs.patch`, for what these options are needed.